io.github.berarma.Oversteer.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: io.github.berarma.Oversteer
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: oversteer
+finish-args:
+  # Wayland support
+  - --socket=wayland
+  # X11 fallback support
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=/run/udev
+  # Access RPM leds
+  - --filesystem=/sys/class/leds
+  # Access the steering wheel parameters
+  - --filesystem=/sys/devices
+  - --device=all
```

Comment:
why id device=all needed?

Reply:
ok

(`input` isn't supported in any stable release, and then it wouldn't work anyway on must users system. There is ongoing work for that).